CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 
 ## Release Notes
 
-No extra steps needed during upgrade.
+- schema migration for cassandra_galley
+- promote stern *after* galley
+- promote spar *after* brig
+- no need to upgrade nginz
```

Comment:
I checked, and the sufficiently general routing pattern is here: https://github.com/zinfra/hegemony/blob/02656750fdecef2e410f7a30fb6254deb4518161/roles/nginz/vars/main.yml#L181